recipes/awkward1/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "awkward1" %}
+{% set version = "0.2.37" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awkward1-{{ version }}.tar.gz
+  sha256: 69a470abe5ac8f9106b90b3753f964ff4654391aecb23ceecac6fffe55c8458b
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py==34]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.13.1
+    - python
+
+test:
+  imports:
+    - awkward1
+    - awkward1._connect
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/scikit-hep/awkward-1.0
+  summary: Manipulate JSON-like data with NumPy-like idioms.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - lgray
+    - chrisburr
+    - jpivarski
```

Comment:
@chrisburr @jpivarski - are you OK with being maintainers?

Reply:
I'm definitely okay with being named a maintainer (I'm the author of the original package), though I'm not familiar enough with Conda build tools to fix a problem if it comes up. If I were the only person to call on, I'd have to scramble to figure this all out.